src/components/00-materials/icons-raw/keyboard_arrow_right.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="currentColor" d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="currentColor" d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"/></svg>
```

Comment:
We have to clarify the usage of the folder "icons-raw". Because @raphaellueckl changed that folder to a "google icons only"-folder. I think we now have a "internal icons" folder - but please ask raphi L where to save these 2 icons...